aiohttp/_websocket/reader_py.py

```diff
@@ -180,173 +185,165 @@ def feed_data(
 
         return EMPTY_FRAME
 
-    def _feed_data(self, data: bytes) -> None:
+    def _handle_frame(
+        self,
+        fin: bool,
+        opcode: Union[int],  # Union is intentional so Cython will convert to C int
```

Comment:
Hmm, what happens when ruff/pyupgrade removes it due to it being deprecated?

Reply:
I'll see if I can come up with something better.  I don't want Cython to complain that its typed to a Python int here but in the pxd typed to a C int.